packages/ckeditor5-ui/src/menubar/utils.ts

```diff
@@ -467,6 +467,30 @@ export const DefaultMenuBarItems: DeepReadonly<MenuBarConfigObject[ 'items' ]> =
 				items: [
 					'menuBar:showBlocks'
 				]
+			},
+			{
+				groupId: 'restrictedEditing',
+				items: [
+					{
+						menuId: 'restrictedEditing',
+						label: 'Restricted editing',
```

Comment:
I think `DefaultMenuBarItems` should be a helper that accepts `Editor` insstance, so we can use `t()` to localize this one. This is the first time we need that but I think it's not reeleasable unless we address this issue.